brands/amenity/bank.json

```diff
@@ -4841,6 +4900,21 @@
       "name": "UCPB"
     }
   },
+  "amenity/bank|UNI": {
+    "countryCodes": ["ca"],
+    "matchNames": [
+      "caisse populaire",
+      "caisse populaire acadienne"
```

Comment:
Just sanity check - is this quite different name intended to match?

Reply:
This was the credit union federation’s name until recently.